spinalcordtoolbox/reports/qc.py

```diff
@@ -337,17 +349,17 @@ def _make_QC_image_for_3d_volumes(self, img, mask, slice_orientation):
         logger.info(self.qc_report.qc_params.abs_bkg_img_path())
         self._save(fig, self.qc_report.qc_params.abs_bkg_img_path(), dpi=self.qc_report.qc_params.dpi)
 
-        for action in self.action_list:
+        fig = Figure()
+        fig.set_size_inches(size_fig[0], size_fig[1], forward=True)
+        FigureCanvas(fig)
+        for i, action in enumerate(self.action_list):
             logger.debug('Action List %s', action.__name__)
             if self._stretch_contrast and action.__name__ in ("no_seg_seg",):
-                print("Mask type %s" % mask.dtype)
-                mask = self._func_stretch_contrast(mask)
-            fig = Figure()
-            fig.set_size_inches(size_fig[0], size_fig[1], forward=True)
-            FigureCanvas(fig)
-            ax = fig.add_axes((0, 0, 1, 1))
-            action(self, mask, ax)
-            self._save(fig, self.qc_report.qc_params.abs_overlay_img_path(), dpi=self.qc_report.qc_params.dpi)
+                print("Mask type %s" % mask[i].dtype)
+                mask[i] = self._func_stretch_contrast(mask[i])
+            ax = fig.add_axes((0, 0, 1, 1), label=str(i))
+            action(self, mask[i], ax)
+        self._save(fig, self.qc_report.qc_params.abs_overlay_img_path(), dpi=self.qc_report.qc_params.dpi)
```

Comment:
I'm wondering if there's a way to make the QC image a little smaller. It's so big that it pushes the text underneath off the page. (See screenshot.)

Reply:
I reduced dpi to 100 instead of 300 as it is done for `sct_label_vertebrae` in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3465/commits/98df805a3bb1488912e37205af85dd75305a9927

Reply:
Much better! 

![Screenshot from 2021-07-21 15-48-52](https://user-images.githubusercontent.com/16181459/126550828-7ffc7e21-e772-43e3-8908-a50fc697089c.png)